src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -21,17 +21,23 @@
 
 .osdQueryEditorFooter {
   color: $euiTextSubduedColor; // Apply the subdued color to all text in this class
-  height: 25px;
+  max-height: 25px;
 
   * {
     color: inherit;
     font-size: $euiFontSizeXS;
     align-items: center;
+    height: 100%;
   }
 }
 
+.osdQueryEditor__filterBarWrapper {
+  margin-top: -15px;
```

Comment:
is this something to do with the gutter?

Reply:
i tried to set gutterSize="none" for EuiFlexGroup but the distance between filter bar and query editor still too far